javascript/javascript1/week1/housePrice.js

```diff
@@ -0,0 +1,33 @@
+//3. Housey pricey (A house price estimator)
+
+// Peter's house parameters
+let peterHouseWidth = 8;
+let peterHouseHeight = 10;
+let peterHouseDepth = 10;
+let peterHouseGardenSize = 100;
+let peterHouseRealPrice = 2500000;
```

Comment:
The variable names are long and descriptive, which is a very good practice. However, you can define them as `const` as they won't change.

Reply:
Changed from let to const...